service/siteService.test.js

```diff
@@ -65,6 +65,42 @@ describe("getMutualAidSites", () => {
     })
 })
 
+describe('getSiteOperationInfo', () => {
+    const openHours = [
+        "recS2hDlNsJgG9Kqn",
+        "recSpjdkWPkmzjc95",
+        "recSpjdkWPkmzjc96",
+        "recS2hDlNsJgG9Kqm",
+        "rect49hsKqi2NLrO1"
+    ]
+    const closeHours = [
+        "recSpjdkWPkmzjc95",
+        "recSpjdkWPkmzjc96",
+        "rect49hsKqi2NLrOx",
+        "rect49hsKqi2NLrOZ",
+        "rect49hsKqi2NLrO1"
+    ]
+
+    test.each`
+        isEnabled | closedDates       | openDates         | openNow  | opening        | closing     
+        ${false}  | ${undefined}      | ${undefined}      | ${"no"}  | ${"never"}     | ${undefined}
+        ${true}   | ${["2021-02-25"]} | ${["2021-02-25"]} | ${"no"}  | ${"not today"} | ${undefined}
+        ${true}   | ${["2021-02-25"]} | ${undefined}      | ${"no"}  | ${"not today"} | ${undefined}
+        ${true}   | ${["2021-02-10"]} | ${["2021-02-05"]} | ${"no"}  | ${"not today"} | ${undefined}
+        ${true}   | ${["2021-02-10"]} | ${undefined}      | ${"yes"} | ${"11:00AM"}   | ${"3:00PM"} 
+        ${true}   | ${["2021-02-10"]} | ${["2021-02-25"]} | ${"yes"} | ${"11:00AM"}   | ${"3:00PM"} 
+        ${true}   | ${undefined}      | ${["2021-02-25"]} | ${"yes"} | ${"11:00AM"}   | ${"3:00PM"} 
+        ${true}   | ${undefined}      | ${undefined}      | ${"yes"} | ${"11:00AM"}   | ${"3:00PM"} 
+    `('should return $opening $closing', ({isEnabled, closedDates, openDates, opening, closing, openNow, expectedSchedule}) => {
+        Settings.now = () => Date.UTC(2021, 1, 25, 17, 10)
+
+        const result = siteService.getSiteOperationInfo(isEnabled, openHours, closeHours, testHours, closedDates, openDates)
```

Comment:
Should I be mocking out the hoursUtils responses here or fine without?

Reply:
I probably wouldn't bother.  The more hardcore would probably do so because of overtesting or some other justification, but in reality I only find it worthwhile to mock if it's:

* A call to something random
* A call to get system time
* An 3rd party / network call
* A DB fetch

Reply:
👍  thanks!